srcpkgs/proton-bridge/patches/Makefile.patch

```diff
@@ -0,0 +1,159 @@
+--- Makefile	2020-05-18 08:02:29.000000000 -0400
++++ "Makefile copy"	2020-06-19 21:28:24.937942084 -0400
```

Comment:
Filenames need to be the same. You should leave a comment on the patch to explain what it's about, btw.

Reply:
> Filenames need to be the same.

Nope, you often see `Makefile` vs ` Makefile~` or so and it works.